crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -706,7 +733,10 @@ fn get_payment_response(
             resource_id: types::ResponseId::ConnectorTransactionId(info_response.id.clone()),
             redirection_data: None,
             mandate_reference: None,
-            connector_metadata: None,
+            connector_metadata: info_response
+                .processor_information
+                .as_ref()
+                .map(|processor_information| serde_json::json!(processor_information.avs)),
```

Comment:
Keep the avs details in "avs_response" key in the metadata

Reply:
Done